### PR TITLE
added test cases for rendering of multiple para and no unsafe tags

### PR DESCRIPTION
**Pull request by prinzpiuz**
test case for no renderning of unsafe tags
and newline create multiple para

### CODE REVIEW

File: homepage/tests.py [hunk 1]
```diff
@@ -72,3 +76,20 @@ def test_post(self):
...
+        self.input1 = {'post_tittle': 'aaaaaaaaaaa', 'post_content': '<h1>will</h1><h2>hi</h2>'}
+
+
+    def test_multiple_para_rendering_and_NO_unsafe_tags(self):
+        response = self.client.post(reverse('homepage:create'), self.input, follow=True)
+        self.assertContains(response, '<p>this tag will not render&lt;a&gt;</p>', status_code=200)
+
+    def test_TOC(self):
+        response = self.client.post(reverse('homepage:create'), self.input1, follow=True)
+        self.assertContains(response, '<ul><li>will</li></ul><ul><ul><li>hi</li></ul></ul>', status_code=200)
```
Comment (aitchnyu) [inline homepage/tests.py]: Fix the code so it needs only 1 `ul` tag for many `li`

File: homepage/tests.py [hunk 2]
```diff
@@ -72,3 +76,20 @@ def test_post(self):
...
+    def setUp(self):
+        self.input = {'post_tittle': 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+                      'post_content': 'this tag will not render<a>\n\nthis is for test'}
+
+        self.input1 = {'post_tittle': 'aaaaaaaaaaa', 'post_content': '<h1>will</h1><h2>hi</h2>'}
+
+
+    def test_multiple_para_rendering_and_NO_unsafe_tags(self):
+        response = self.client.post(reverse('homepage:create'), self.input, follow=True)
+        self.assertContains(response, '<p>this tag will not render&lt;a&gt;</p>', status_code=200)
```
Comment (aitchnyu) [inline homepage/tests.py]: Test the second line is generated in another paragraph

File: homepage/tests.py (hunk 1 above)
Comment (prinzpiuz) [reply homepage/tests.py]: rendering is not working/looking fine when doing like that

File: homepage/templates/homepage/login.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+{% if form.errors %}
+<p>Your username and password didn't match. Please try again.</p>
```
Comment (aitchnyu) [inline homepage/templates/homepage/login.html]: Fix Indenatation problems

**Pull request merged**